doc/manual/src/hacking.md

```diff
@@ -23,7 +17,12 @@ To build Hydra, you should then do:
     [nix-shell]$ configurePhase
     [nix-shell]$ make
 
-You can run the Hydra web server in your source tree as follows:
+You start a local database, the webserver, and other components with
+foreman:
+
+    $ foreman start
+
```

Comment:
```suggestion

```ShellSession
$ foreman start
```

```

That should improve highlighting on github and maybe mdbook.